### PR TITLE
Rename test-spring-security-helper -> test-spring-security-flow

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

**Comment by knoobie** [comment]
Should `fusion` be excluded from the project to showcase that this is flow only and works without `fusion` as dependency?

**Comment by Artur-** [comment]
It is actually excluded right now as `fusion-endpoint` is set as provided in the `vaadin-spring` pom.xml.

**Comment by knoobie** [comment]
You are right, `vaadin-spring` doesn't use `vaadin-core` and defines all dependencies by itself.

**Review by haijian-vaadin — DISMISSED** [review]

File: vaadin-spring-tests/test-spring-security-flow/pom.xml [hunk 1]
```diff
@@ -7,8 +7,8 @@
         <artifactId>vaadin-spring-tests</artifactId>
         <version>17.0-SNAPSHOT</version>
     </parent>
-    <artifactId>test-spring-secturity-helper</artifactId>
-    <name>Integration tests for Vaadin Spring Security helper</name>
+    <artifactId>test-spring-secturity-flow</artifactId>
```
Comment (joheriks) [inline vaadin-spring-tests/test-spring-security-flow/pom.xml:10]: Noticed a typo in the original artifactId (`secturity`) that perhaps could be fixed in this change as well.

Comment (Artur-) [reply vaadin-spring-tests/test-spring-security-flow/pom.xml]: Ifxed teh ypto

**Review by joheriks — APPROVED** [review]

**Pull request merged**